system-variables.md

```diff
@@ -104,6 +104,12 @@ mysql> SELECT * FROM t1;
 - 默认值：（系统主机名）
 - 这个变量一个只读变量，表示 TiDB server 的主机名。
 
+### `init_connect`
+
+- 作用域：GLOBAL
+- 默认值：""
+- `init_connect` 特性允许首次连接到 TiDB 服务器时自动执行 SQL 语句。如果你有 `CONNECTION_ADMIN` 或者 `SUPER` 权限，此条 `init_connect` 语句将不会被执行。如果 `init_connect` 语句报错，你的用户连接将被终止。
```

Comment:
@xhebox PTAL whether this comment is correct. Thanks!
```suggestion
- 用户首次连接到 TiDB 服务器时，`init_connect` 特性允许 TiDB 自动执行一条 SQL 语句。如果你有 `CONNECTION_ADMIN` 或者 `SUPER` 权限，此条 SQL 语句将不会被自动执行。如果此条语句执行报错，你的用户连接将被终止。
```

Reply:
I think it is not limited to only one stmt.

Reply:
@xhebox I've changed my comment according to your suggestion. PTAL again, thanks!